src/bm_sim/P4Objects.cpp

```diff
@@ -2499,6 +2513,11 @@ P4Objects::check_hash(const std::string &name) const {
   return nullptr;
 }
 
+bool P4Objects::is_selector_fanout(const Json::Value &cfg_selector) const {
+  return cfg_selector.isMember("algo") && // not to hardcode names, fix later
```

Comment:
Remove comment `// not to hardcode names, fix later`. Seems as though hardcoding is fine in this case as it follows the convention of all the other code like the `process_cfg_selector` function.